logstash-core/src/test/java/org/logstash/secret/store/SecretStoreFactoryTest.java

```diff
@@ -4,17 +4,13 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.rules.TemporaryFolder;
-import org.logstash.secret.EnvironmentUtil;
 import org.logstash.secret.SecretIdentifier;
 import org.logstash.secret.store.backend.JavaKeyStore;
 
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
nitpick: i think we have been avoiding wildcard imports.

Reply:
👍 

Side note: we really should have a shared IntelliJ config then -- the default config rewrites with wildcards any time we end up with 5 or more individual imports.